tests/test_cfg_end_to_end_tests.py

```diff
@@ -175,99 +84,72 @@ def get_test_cases():
-            os.makedirs(os.path.dirname(obj.key))
-        bucket.download_file(obj.key, obj.key)
-    print("Test data downloaded to 'test_data' folder.")
+    logging.info(f"Downloading test data for {rel_path_from_root} from s3")
+    
+    prefix = rel_path_from_root + "/"
+    for obj in bucket.objects.filter(Prefix=prefix):
+        local_file_path = Path("test_data") / obj.key
+        local_file_path.parent.mkdir(parents=True, exist_ok=True)
+        bucket.download_file(obj.key, str(local_file_path))
+    
+    logging.info(f"Test data downloaded to 'test_data/{rel_path_from_root}' folder.")
     os.environ["TEST_DATA_ROOT"] = os.path.abspath("test_data")
-
     return os.environ["TEST_DATA_ROOT"]
 
-
 @pytest.mark.skipif(
     not check_e2e_test_data(),
     reason="Either TEST_DATA_ROOT needs to be defined or both AWS_SECRET_KEY "
     "and AWS_ACCESS_KEY to run e2e config tests",
 )
 @pytest.mark.parametrize("config_path,data_check_fn", get_test_cases())
-def test_configs(config_path: str, data_check_fn: Callable, tmp_path: str):
-    test_data_root = get_e2e_test_data_path()
-    # we expect DATASET_CONFIGS_ROOT and TEST_DATA_ROOT
-    # to have the same structure (e.g. <lang>/<dataset>)
-    rel_path_from_root = os.path.relpath(Path(config_path).parent, DATASET_CONFIGS_ROOT)
-
-    # run data_check_fn - it will raise error if the expected test data is not found
-    data_check_fn(raw_data_dir=str(Path(test_data_root) / rel_path_from_root))
+def test_configs(config_path: str, data_check_fn: Callable, tmp_path: Path):
+    rel_path_from_root = Path(config_path).parent.relative_to(DATASET_CONFIGS_ROOT)
+    test_data_root = Path(get_e2e_test_data_path(str(rel_path_from_root)))
+    data_check_fn(raw_data_dir=str(test_data_root / rel_path_from_root))
 
-    reference_manifest = str(Path(test_data_root) / rel_path_from_root / "test_data_reference.json")
-    if not os.path.exists(reference_manifest):
+    reference_manifest = test_data_root / rel_path_from_root / "test_data_reference.json"
+    if not reference_manifest.exists():
         raise ValueError(f"Did not find reference manifest {reference_manifest}")
 
     cfg = OmegaConf.load(config_path)
     assert "processors" in cfg
-    cfg["processors_to_run"] = "all"
-    cfg["workspace_dir"] = str(tmp_path)
-    cfg["final_manifest"] = str(tmp_path / "final_manifest.json")
-    if "data_split" not in cfg:
-        cfg["data_split"] = "train"
-    cfg["processors"][0]["raw_data_dir"] = str(Path(test_data_root) / rel_path_from_root)
+    cfg.processors_to_run = "all"
+    cfg.workspace_dir = str(tmp_path)
+    cfg.final_manifest = str(tmp_path / "final_manifest.json")
+    cfg.data_split = cfg.get("data_split", "train")
+    cfg.processors[0].raw_data_dir = str(test_data_root / rel_path_from_root)
 
     run_processors(cfg)
-    # additionally, let's test that final generated manifest matches the
-    # reference file (ignoring the file paths)
```

Comment:
let's add back the comment

Reply:
Just trying to fix ci/cd, this will be reverted, no worries